enterprise/backend/test/metabase_enterprise/serialization/cmd_test.clj

```diff
@@ -22,7 +22,7 @@
   what we need."
   [& body]
   `(schema-migrations-test.impl/with-temp-empty-app-db [conn# :h2]
-     (schema-migrations-test.impl/run-migrations-in-range! conn# [0 999999]) ; this should catch all migrations)
+     (schema-migrations-test.impl/run-migrations-in-range! conn# [0 "v99.00-000"]) ; this should catch all migrations)
```

Comment:
this should have been in the PR to switch to the new migration format but I missed this